extra_data/export.py

```diff
@@ -183,44 +130,43 @@ def feed(self, data, metadata=None):
         ----------
         data : dict
             Contains train data. The dictionary has to follow the karabo_bridge
-            protocol structure:
-
-            - keys are source names
-            - values are dict, where the keys are the parameter names and
-              values must be python built-in types or numpy.ndarray.
+            see :func:`~karabo_bridge.serializer.serialize` for details
 
         metadata : dict, optional
-            Contains train metadata. The dictionary has to follow the
-            karabo_bridge protocol structure:
-
-            - keys are (str) source names
-            - values (dict) should contain the following items:
-
-              - 'timestamp' Unix time with subsecond resolution
-              - 'timestamp.sec' Unix time with second resolution
-              - 'timestamp.frac' fractional part with attosecond resolution
-              - 'timestamp.tid' is European XFEL train unique ID
-
-            ::
-
-              {
-                  'source': 'sourceName'  # str
-                  'timestamp': 1234.567890  # float
-                  'timestamp.sec': '1234'  # str
-                  'timestamp.frac': '567890000000000000'  # str
-                  'timestamp.tid': 1234567890  # int
-              }
-
-            If the metadata dict is not provided it will be extracted from
-            'data' or an empty dict if 'metadata' key is missing from a data
-            source.
+            Contains train metadata. If the metadata dict is not provided it
+            will be extracted from 'data' or an empty dict if 'metadata' key
+            is missing from a data source.
+            see :func:`~karabo_bridge.serializer.serialize` for details
```

Comment:
If we put this function in Sphinx docs somewhere, we can configure [Intersphinx](https://docs.readthedocs.io/en/stable/guides/intersphinx.html) to cross-link to it.

In either case, I think we should drop the `~`, which shows only the last part of the name in the docs. Showing the full name makes it clear that it's in a separate package.

Reply:
Right. The documentation for the karabo bridge project isn't actually self contained (and outdated in user-documentation...). I'll look if that makes sens to change that.